javascripts/2048.js

```diff
@@ -41,18 +47,15 @@ function newTile(openCells, usedCells) {
   $(tile).attr("data-val", initNum);
   $(tile).text(initNum);
   occupyCell(openCells, usedCells, cellSpace);
-  // emptyCell(openCells, usedCells, cellSpace)
-
 }
 
 function occupyCell(openCells, usedCells, cellSpace) {
   var index = findIndex(openCells, cellSpace);
   openCells.splice(index, 1);
-  var cellExists = findIndex(usedCells, cellSpace)
+  var cellExists = findIndex(usedCells, cellSpace);
   if (cellExists == undefined) {
```

Comment:
Is this check needed? The way you're tracking open and used cells should make it so you never be able to push a used celled into usedCells.